performance_visualization.py

```diff
@@ -0,0 +1,33 @@
+import json
```

Comment:
No sure of adding this as the scope of llm-load-test, can you push this in another PR for for further discussion?

Reply:
It's already used in utils.py, and it's a common package

Reply:
nana I mean the visualization script, so we can merge this before and talk about the visualization later

Reply:
Ah! Sorry for misunderstanding, I agree. @HunterGerlach can we merge this PR without the visualization code, and have a separate PR for that?

Reply:
Absolutely. I actually hesitated including it initially, so that's fine.